config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :flats, only: [ :index, :show, :create, :new ] do
+    resources :bookings, only: [ :new, :create ]
```

Comment:
@ropicaza carefull because you are adding `/flats/:flat_id/bookings/new` by adding `:new`

Reply:
@Joacocasanova resolve by #2{b1bd99f}